Dockerfile

```diff
@@ -0,0 +1,44 @@
+#FROM ocm-container
+FROM fedora:latest
+
+ENV I_AM_IN_CONTAINER="I-am-in-container"
+
+RUN yum -y install \
```

Comment:
`yum` is deprecated/deprecating in current versions of Fedora .  Perhaps change to `dnf`?

Reply:
I believe we are going to leave `fedora` as a base image anyway and either move to coreos container linux or centos as our base, but I don't think we've finalized on which we are moving towards.  Good to keep in mind though if something breaks before we make the migration.

Reply:
:+1:

Reply:
I think it might be good to deploy multiple base image versions... `ocm-container:fedora`, `om-container:ubuntu`, `ocm-container:alpine` for example. 

As we use it as an interactive environment, people shouldn't be made to learn tools they're not familiar with.